components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityConstants.java

```diff
@@ -80,6 +80,7 @@
 
     public static final int API_AUTH_MISSING_OPEN_API_DEF = 900918;
     public static final String API_AUTH_MISSING_OPEN_API_DEF_ERROR_MESSAGE = "Internal Server Error";
+    public static final String GRAPHQL_API_FAILURE_HANDLER = "_graphql_failure_handler";
```

Comment:
Have we tracked this anywhere in a documentation?

Reply:
will track